blockly-core/core/ui/block.js

```diff
@@ -558,6 +558,14 @@ Blockly.Block.prototype.getBox = function() {
 };
 
 /**
+ * Returns the padding of the SVG or null if none exists
+ * @return {Object} object with padding values for top, bottom, left, and right
+ */
+Blockly.Block.prototype.getSvgPadding = function() {
+  return this.svg_ && this.svg_.getPadding();
```

Comment:
Better to return `{top: 0, right: 0, bottom: 0, left: 0}` vs. `false` when `this.svg_` is null?  `padding.*` is referenced below without checking if `padding` is null.